text/0008-sync-and-storage-review-packet.md

```diff
@@ -0,0 +1,371 @@
+# Roadmap Review: Sync and Storage
+
+**Firefox should wow users with a smart personalized experience that's available seamlessly across services, apps, and devices. User data storage and syncing are vital to this vision.**
+
+We propose to reframe user profile data as a pillar of the Firefox ecosystem, rather than continuing to allow our storage and syncing story to emerge piecemeal from the implementation details of each new product feature.
+
+User data becomes a deliberately designed service layer on which many different product experiences are built. We foster through design reviews the long-term importance of user data. And we invest technically in a more unified, extensible, flexible, cross-platform suite of storage and sync solutions that support a coherent set of approaches to storage.
+
+## Review question
+
+Is this roadmap proposal congruent with Firefox's strategy?
+
+## Document status
+
+*   2017-11-20: Go-ahead given.
+*   2017-11-09: Review successfully completed. Follow-up questions underway.
+*   2017-11-03: Sufficient contributions received; sent to the Reviewer.
+
+## Roles
+
+<dl>
```

Comment:
Could we avoid dropping down to HTML here?:

```
* **Reviewer**: dcamp
* **Chair**: jwalker
* **Proposers**: rnewman (Browser Architecture), rfkelly (Engineering Lead, Firefox Accounts), markh (Sync engineering), emily (Browser Architecture), adavis (Product Manager, Sync & FxA)
```

Reply:
*shakes fist at Markdown*